digiwf-integrations/digiwf-s3-integration/digiwf-s3-integration-core/src/main/java/io/muenchendigital/digiwf/s3/integration/adapter/in/rest/FolderController.java

```diff
@@ -28,16 +26,16 @@
 @RequestMapping("/folder")
 public class FolderController {
 
-    private final FolderHandlingService folderHandlingService;
 
+    private final FolderOperationsInPort folderOperations;
     private final FilesInFolderMapper filesInFolderMapper;
 
     @DeleteMapping
     @Operation(description = "Deletes the folder specified in the parameter together with the corresponding database entry")
-    public ResponseEntity<Void> delete(@RequestParam @NotEmpty @Size(max = FileRepository.LENGTH_PATH_TO_FILE) final String pathToFolder) {
+    public ResponseEntity<Void> delete(@RequestParam @NotEmpty final String pathToFolder) {
```

Comment:
We do not use this endpoint at the moment - but would not delete it. We would have to ask some other projects that are using this lib.

Reply:
Ok. No changes for the moment.